manuscript.md

```diff
@@ -357,7 +349,41 @@ complexity regardless of connectance.
 
 ## Larger networks are less complex than they could be
 
-
+To assess whether ecological networks are more, or less, complex than expected,
+we applied two null models that generate pseudo-random networks: Type I
+[@Fortuna2006HabLosa], where interactions happen proportionally to connectance,
+and Type II [@Bascompte2003NesAss], where interactions happen proportionally to
+the joint degree of the two species involve. We generated 999 samples for every
```

Comment:
I wonder whether it may be appropriate to reference the more "classic" random graphs names for these models (Erdös-Renyi Gnp and Configuration model) here. This would suggest a link to a throve of established results...

Reply:
I agree - I am quite convinced that the ecologists have re-discovered them by accident. What would you cite as the reference for each? I would dig up my copy of Newman but I'm not sure where it is...